docs/DeveloperGuide.adoc

```diff
@@ -334,21 +339,78 @@ Use case ends.
 +
 Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. The given eatery is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. EatMe shows an error message.
 +
 Use case resumes at step 2.
 
+* 6a. The given information is of the wrong format.
++
+[none]
+** 6a1. EatMe shows an error message.
++
+Use case resumes at step 8.
+
+[discrete]
+=== Use case: Tagging an eatery in main
+
+*MSS*
+
+1.  User requests to add a tag to an eatery
+2.  EatMe updates the eatery accordingly
++
+Use case ends.
+
+*Extensions*
```

Comment:
You may want to format the nesting of your **2a. The List is empty**, **Use case ends.**.